src/contracts/account/ReplayProtection.sol

```diff
@@ -53,15 +42,17 @@ contract ReplayProtection {
         address signer = verifySig(_callData, _replayProtection, _replayProtectionAuthority, getChainID(), _signature);
 
         // Check the user's replay protection.
-        if(_replayProtectionAuthority == getMultiNonceAddress()) {
+        if(_replayProtectionAuthority == multiNonceAddress) {
             // Assumes authority returns true or false. It may also revert.
             require(nonce(signer, _replayProtection), "Multinonce replay protection failed");
-        } else if (_replayProtectionAuthority == getBitFlipAddress()) {
+        } else if (_replayProtectionAuthority == bitFlipAddress) {
             require(bitflip(signer, _replayProtection), "Bitflip replay protection failed");
         } else {
             require(IReplayProtectionAuthority(_replayProtectionAuthority).updateFor(signer, _replayProtection), "Replay protection from authority failed");
         }
 
+        emit ReplayProtectionInfo(_replayProtectionAuthority, _replayProtection);
```

Comment:
I think it would good to somehow include the hash of the encoded data in an event, either this one or a new one. Since this may be the only way for a user to confirm that their tx was delivered? If we do include it, it should be `indexed`

Reply:
I've added a test in replayprotection.test.ts that covers this